CHANGELOG.md

```diff
@@ -10,6 +10,12 @@ and this project adheres to
 
 + No changes yet
 
+## 15.0.2 - 2020-06-15
```

Comment:
[Releasing happens separately](https://github.com/uPortal-Project/uportal-app-framework/blob/master/docs/releasing.md) from feature branches in this project, so it's safer to not anticipate that the release will be that version on this date. Might be. Might not be. No need to commit to it in this changeset.

Reply:
Thanks Andrew! I removed those dates.